subsys/bluetooth/audio/unicast_client.c

```diff
@@ -1908,6 +1911,204 @@ static int unicast_client_ase_discover(struct bt_conn *conn,
 	return bt_gatt_read(conn, &params->read);
 }
 
+static uint8_t unicast_client_pacs_avail_ctx_read_func(struct bt_conn *conn,
+						      uint8_t err,
+						      struct bt_gatt_read_params *read,
+						      const void *data,
+						      uint16_t length)
+{
+	struct bt_audio_discover_params *params;
+	struct bt_pacs_context context;
+	struct net_buf_simple buf;
+
+	params = CONTAINER_OF(read, struct bt_audio_discover_params, read);
+
+	BT_DBG("conn %p err 0x%02x len %u", conn, err, length);
+
+	/* Avail_ctx is an optional characteristic and may not exist; if not we
```

Comment:
This statement is not valid. Available Audio Contexts is mandatory in PACS v.1.0.1

Reply:
Too much copy-paste - Thanks! :)